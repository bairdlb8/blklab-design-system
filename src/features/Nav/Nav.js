import React from 'react'
import PropTypes from 'prop-types'

import './Nav.scss'

const Nav = ({className=''}) => {
  return <div>Nav</div>
}

Nav.propTypes = {}

export default Nav
