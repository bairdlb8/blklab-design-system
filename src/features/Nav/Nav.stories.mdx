import { Meta, Story, Props, Preview } from '@storybook/addon-docs/blocks'

import Nav from './Nav'

<Meta title="Features / Nav" component={ Nav } />

# Nav

## Add some text here

export const Template = (args) => {
  return <Nav {...args} />
}

<Preview withToolbar>
  <Story name="Primary" args={ {} }>
    {Template.bind({})}
  </Story>
</Preview>

<Props of={ Nav } />
