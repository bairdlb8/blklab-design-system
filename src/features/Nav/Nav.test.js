import React from 'react'
import { render } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import { axe, toHaveNoViolations } from 'jest-axe'
import Nav from './Nav'

expect.extend(toHaveNoViolations)

describe('<Nav />', () => {
  describe('accessibility', () => {
    it('should not fail any accessibility tests', async () => {
      const { container } = render(<Nav />)
      expect(await axe(container)).toHaveNoViolations()
    })
  })

  it('should take a snapshot', () => {
    const { asFragment } = render(<Nav />)

    expect(asFragment(<Nav />)).toMatchSnapshot()
  })
})
