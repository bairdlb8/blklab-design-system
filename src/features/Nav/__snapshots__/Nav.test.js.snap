// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Nav /> should take a snapshot 1`] = `
<DocumentFragment>
  <div>
    Nav
  </div>
</DocumentFragment>
`;
